### PR TITLE
Fastai test 

### PR DESCRIPTION
**Pull request by jjmachan**
## Description

Adds integration test for fastai2 lib
## Motivation and Context

based on #1194 

## How Has This Been Tested?

I've tested I locally, only issue is that there is a model.pkl file being generated by I'm still confused as to where its coming from.

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature and improvements (non-breaking change which adds/improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code Refactoring (internal change which is not user facing)
- [ ] Documentation
- [x] Test, CI, or build

## Component(s) if applicable
- [ ] BentoService (service definition, dependency management, API input/output adapters)
- [ ] Model Artifact (model serialization, multi-framework support)
- [ ] Model Server (mico-batching, dockerisation, logging, OpenAPI, instruments)
- [ ] YataiService gRPC server (model registry, cloud deployment automation)
- [ ] YataiService web server (nodejs HTTP server and web UI)
- [ ] Internal (BentoML's own configuration, logging, utility, exception handling)
- [ ] BentoML CLI

## Checklist:

- [x] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [x] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: ci/fastai_integration_tests.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# https://stackoverflow.com/questions/42218009/how-to-tell-if-any-command-in-bash-script-failed-non-zero-exit-status/42219754#42219754
+# Set err to 1 if pytest failed.
+error=0
+trap 'error=1' ERR
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+cd "$GIT_ROOT" || exit
+
+# Install PyTorch
+pip install fastai==2.2.5
```
Comment (yubozhao) [inline ci/fastai_integration_tests.sh:13]: Do we need to pin the version here?

Comment (jjmachan) [reply ci/fastai_integration_tests.sh:13]: I guess not, Fastai 2 is the default now rt?

Comment (yubozhao) [reply ci/fastai_integration_tests.sh:13]: yep

**Comment by yubozhao** [comment]
@jjmachan Thank you for doing the heavy lifting for this. I went ahead updated the version and merged it.

**Pull request merged**